### PR TITLE
Add logic for multiremote object when setting tunnelIdentifier

### LINKED ISSUE #41: Setting tunnel identifier fails when using multiremote
## The problem

When using multiremote and attempting to set a tunnel identifier in the wdio config file, the following error occurs:

```
A service failed in the 'onPrepare' hook
TypeError: capabilities.forEach is not a function
    at SauceLaunchService.onPrepare (/Users/momaney/Documents/react-ciscospark/node_modules/wdio-sauce-service/build/sauce-launch-service.js:49:30)
    at /Users/momaney/Documents/react-ciscospark/node_modules/webdriverio/build/lib/launcher.js:406:66
    at Array.map (native)
    at Launcher._callee2$ (/Users/momaney/Documents/react-ciscospark/node_modules/webdriverio/build/lib/launcher.js:404:71)
    at tryCatch (/Users/momaney/Documents/react-ciscospark/node_modules/regenerator-runtime/runtime.js:65:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/momaney/Documents/react-ciscospark/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/momaney/Documents/react-ciscospark/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/Users/momaney/Documents/react-ciscospark/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /Users/momaney/Documents/react-ciscospark/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at Promise.F (/Users/momaney/Documents/react-ciscospark/node_modules/core-js/library/modules/_export.js:35:28)

Continue...
ERROR: Sauce could not start your job. For more information on what happened, please visit https://saucelabs.com/jobs/02efeb126e0b4662b00b9117baab86d8

    no stack available
ERROR: Sauce could not start your job. For more information on what happened, please visit https://saucelabs.com/jobs/02efeb126e0b4662b00b9117baab86d8

    at new RuntimeError (/Users/momaney/Documents/react-ciscospark/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:144:12)
    at Request._callback (/Users/momaney/Documents/react-ciscospark/node_modules/webdriverio/build/lib/utils/RequestHandler.js:313:39)
    at Request.self.callback (/Users/momaney/Documents/react-ciscospark/node_modules/request/request.js:186:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/Users/momaney/Documents/react-ciscospark/node_modules/request/request.js:1163:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Users/momaney/Documents/react-ciscospark/node_modules/request/request.js:1085:12)
    at IncomingMessage.g (events.js:291:16)
```

## Environment

* wdio-sauce-service version: 0.4.4
* webdriverio version: 4.9.11
* Node.js version: v6.9.1
* [Standalone mode or wdio testrunner](http://webdriver.io/guide/getstarted/modes.html): wdio testrunner

### PR DESCRIPTION
**Pull request by moriahmaney**
Would close [issue #41](https://github.com/webdriverio/wdio-sauce-service/issues/41)

### CODE REVIEW

File: lib/sauce-launch-service.js [hunk 1]
```diff
@@ -21,9 +21,17 @@ class SauceLaunchService {
...
-            capabilities.forEach((capability) => {
-                capability.tunnelIdentifier = capability.tunnelIdentifier || sauceConnectTunnelIdentifier
-            })
+            if (Array.isArray(capabilities)) {
+                capabilities.forEach((capability) => {
+                    capability.tunnelIdentifier = capability.tunnelIdentifier || sauceConnectTunnelIdentifier
+                })
+            } else {
+                Object.keys(capabilities).forEach((browser) => {
+                    if (capabilities[browser].desiredCapabilities) {
```
Comment (christian-bromann) [inline lib/sauce-launch-service.js]: Why should `capabilities[browser].desiredCapabilities` ever be undefined?

Comment (moriahmaney) [reply lib/sauce-launch-service.js]: I suppose it shouldn't, had added it just to be safe. Can remove if you prefer.

Comment (christian-bromann) [reply lib/sauce-launch-service.js]: > Can remove if you prefer.

Yes, if we don't expect this to be undefined let not check for it

Comment (moriahmaney) [reply lib/sauce-launch-service.js]: Sure, removed it.

**Review by christian-bromann — APPROVED** [review]
Thanks, lgtm 👍

**Comment by christian-bromann** [comment]
wdio-sauce-service 0.4.4 published 🎉

**Comment by moriahmaney** [comment]
Thanks @christian-bromann 😊 🎉

**Comment by moriahmaney** [comment]
hmmm @christian-bromann i downloaded the latest version available 0.4.5 and it seems that the code from this pr isn't in there.

**Comment by christian-bromann** [comment]
Lemme check

**Comment by christian-bromann** [comment]
@moriahmaney something was odd with my build, rereleased it, please check

**Comment by moriahmaney** [comment]
@christian-bromann working now, thanks!

**Pull request merged**